src/emmc.c

```diff
@@ -252,7 +252,7 @@ pu_emmc_write_data(PuFlash *flash,
             continue;
         }
 
-        part_mount = pu_create_mount_point(g_strdup_printf("p%u", i));
+        part_mount = pu_create_mount_point(g_strdup_printf("p%u", i), error);
         if (part_mount == NULL) {
             g_warning("Failed creating mount point");
```

Comment:
I'm unsure how to handle an error at this point, because the current implementation simply ignores failure and goes on to the next partition. Should the error be printed and cleared or should the function return?

Reply:
I think we should be more strict at this point and return an error instead of handling it only as a warning. In most cases, failing to create a mount point indicates some real issue and it should be handled as such. Further operations might fail anyways, if there is no mount point.